### PR TITLE
feat(docs-infra): redesign contributors page

### LINKED ISSUE #44346: Angular Contributors page focus lost after clicking on a card
### Description

Angular Contributors page using a pattern where cards are displayed which flip over when clicking on them. this creates a11y problems because focus is lost when flipping the card. Also, this is explicitly not allowed in the material design spec.

### What is the affected URL?

https://angular.io/about?group=Angular

### Please provide the steps to reproduce the issue

1. navigate to https://angular.io/about?group=Angular
2. press tab until focused on "Alan Agius" card
3. press space
4 (Alan's card flips over)
5. press space again
6. browser scrolls down on the page

### Please provide the expected behavior vs the actual behavior you encountered

this pattern of having cards that flip should not be used at all. Focus show not be lost after clicking on a card.

### Please provide a screenshot if possible

_No response_

### Please provide the exception or error you saw

_No response_

### Is this a browser-specific issue? If so, please specify the device, browser, and version.

_No response_

### PR DESCRIPTION
**Pull request by dario-piotrowicz**
redesign the aio contributors page so that it displays the contributors
details as a list instead of using flipping cards, this should improve
the usability of the page and especially its accessibility

resolves #44346

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #44346

This is the current page:
before

## What is the new behavior?

This is how it looks after the changes:
after

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
 - I made sure to make the page responsive both on the window's size and on the browser's font-size :slightly_smiling_face: 
 - I have checked caniuse and I unless I missed something all the css properties I have used should be supported by all modern browsers

### CODE REVIEW

File: goldens/size-tracking/aio-payloads.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
       "runtime": 4325,
       "main": 456483,
       "polyfills": 33814,
-      "styles": 72183,
+      "styles": 71574,
```
Comment (dario-piotrowicz) [inline goldens/size-tracking/aio-payloads.json:7]: @gkalpak :grin::+1:

**Review by jessicajaniuk — APPROVED** [review]
Drive by review! I love this change! LGTM!!!

**Comment by zarend** [comment]
Excellent improvement ✔️  cc @twerske

**Review by zarend — APPROVED** [review]

**Comment by zarend** [comment]
This looks good to me. @AndrewKushnir should the commit header be "fix(docs-infra)"?

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit 7558c558e051a787a953b63f73b73f2288cc6143.

**Pull request closed without merging**